exporters/metric/cortex/cortex.go

```diff
@@ -347,32 +278,32 @@ func convertFromHistogram(record metric.Record, histogram aggregation.Histogram)
 		return nil, err
 	}
 
-	var totalCount float64
+	var totalCount int64
 	// counts maps from the bucket upper-bound to the cumulative count.
 	// The bucket with upper-bound +inf is not included.
-	counts := make(map[float64]float64, len(buckets.Boundaries))
+	counts := make(map[float64]int64, len(buckets.Boundaries))
```

Comment:
Let's leave these float64s (changed in this file) as float64s. Even though the internal structures changed, the output is going to be coerced back to float in `createTimeSeries`.